static/js/yucisite.js

```diff
@@ -25,3 +25,12 @@ $("#menuIconOpen").click(function() {
   $(".login").show();
   $("#menuIconClosed").show();
 });
+
+$("#name").click(function() {
+  var classes = $(".profile").attr('class')
```

Comment:
You can also use `toggleClass` ([read more here](http://api.jquery.com/toggleclass/)).

```javascript
$("#name").click(function() {
  $('.profile').toggleClass('hidden');
});
```